src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineFunctions.scala

```diff
@@ -33,7 +33,7 @@ object PipelineFunctions {
       case None =>
         connectionString
     }
-    if (!retrievedConnectionString.matches("^Endpoint=sb://[a-zA-Z0-9-_:;/.=]*;SharedAccessKey=[a-zA-Z0-9-_:;/.=]*$")) {
+    if (!retrievedConnectionString.matches("^Endpoint=sb://[a-zA-Z0-9-_:;/.=]*;SharedAccessKey=[a-zA-Z0-9-_:;/.=+]*$")) {
```

Comment:
we can potentially simplify regex to `^Endpoint=sb://.*;SharedAccessKey=.*$` to avoid additional validation problems

Reply:
done, was considering doing this. thanks